### PR TITLE
Use Eclipse branding and trademarks

### PR DESCRIPTION
**Pull request by mwperina**
- **Use standard instead of alternate markdown syntax**
- **Use project formal name in first and all prominent references**
- **Add social links to the footer**
- **Add links required by Eclipse to website footer**
- **Add trade mark to man pages**

### CODE REVIEW

**Comment by mwperina** [comment]
Here is the screenshot showing documentation website changes
Screenshot from 2024-04-09 13-33-48

File: doc/README.md [hunk 1]
```diff
@@ -1,48 +1,45 @@
-BlueChi's documentation pages
-===========================
+# BlueChi's documentation pages
```
Comment (engelmi) [inline doc/README.md]: Here the trademark isn't needed since its about `BlueChi`, not the project Eclipse BlueChi, right?

**Review by engelmi — COMMENTED** [review]
Does it need a trademark in the `CONTRIBUTING.md` as well? It is based on an eclipse template with no trademark, but it would make sense, I think.

File: doc/README.md (hunk 1 above)
Comment (mwperina) [reply doc/README.md]: I've added the trade mark to both files, thanks

**Review by engelmi — APPROVED** [review]
LGTM

**Pull request merged**